### PR TITLE
Berlin review 

### PR DESCRIPTION
**Pull request by vblagoje**
Contains suggestions/fixes from Berlin review with Pedro (except configuration fixes). Please have a look @tristantarrant

### CODE REVIEW

File: src/main/webapp/cluster-nodes/cluster-nodes.html [hunk 1]
```diff
@@ -75,7 +75,7 @@ <h2 class="card-pf-title">Add Node
             inetAddress:server.getInetAddress()})">
           <div class="card-pf card-pf-accented card-pf-aggregate-status card-pf-with-action">
             <h2 class="card-pf-title">
-              <span class="pficon pficon-ok"></span>{{server.name}}
+              <span class="pficon" ng-class="server.isStopped()?'pficon-warning-triangle-o':'pficon-ok'"></span>{{server.name}}
```
Comment (tristantarrant) [inline src/main/webapp/cluster-nodes/cluster-nodes.html]: let's use fa-power-off instead of pficon-warning-triangle-o

Comment (vblagoje) [inline src/main/webapp/cluster-nodes/cluster-nodes.html]: Right, I missed that one!

Comment (vblagoje) [inline src/main/webapp/cluster-nodes/cluster-nodes.html]: @tristantarrant updated. Have a look again. Looks much better now!

Comment (tristantarrant) [inline src/main/webapp/cluster-nodes/cluster-nodes.html]: Ahem you did not push

**Pull request merged**